_posts/2016-09-22-maturing-the-security-project.markdown

```diff
@@ -0,0 +1,162 @@
+the level of engagement we have in the community, after all, emails were sent on
+-dev to highlight the closing nomination window, I even received them as you can see below (It's the one on the 19th)...
+
+![PTL election reminder](../assets/SecurityMail.png)
+
+Unfortunately during the nomination window I was temporarily distracted dealing with
+some local issues. I've discussed these with a member of the TC who recognises that
+it was a temporary thing that's unlikely to happen in the future, however the bell
+has been rung and we must decide how to proceed.
+
+## Maturing the Security Project
+Missing two nominations reflects badly on a project team and leads to several understandable
+[questions](http://lists.openstack.org/pipermail/openstack-dev/2016-September/104170.html) being
+asked: _who are these guys?_ _are they an active team?_ _should they be moved outside of the big
+tent?_
+
+These are understandable questions, I feel that my [on-thread
+response](http://lists.openstack.org/pipermail/openstack-dev/2016-September/104176.html) addressed
+them for the most part. What I want to focus on in this blog post is the things that we need to do to
+be a better part of the community. To ensure that project teams and the TC are both aware of what we
+do and how we help improve security in OpenStack.
+
+We know from the feedback we've had from downstream OpenStack consumers that our work
+is valued, we need to better demonstrate that value within the OpenStack community. I think a good
+place to start is to look at the [Project Team Guide](http://docs.openstack.org/project-team-guide)
+examine what we are already doing and where we fall short. Of course this doesn't include the good
+things we do like providing CI tooling for security, threat analysis etc but it is the minimum boxes
+that we should be ticking off as a project team and that I should be driving as PTL.
+
+I want to be clear, I think that the Security Project is doing great things to enhance security
+in OpenStack. We need to become a better community player though, through doing so I expect
+new opportunities to innovate on security and create new ways to make OpenStack more secure.
+
+## Score Card
+I'm sure this isn't a new idea but I wasn't able to find much good evidence of
+other teams doing this when I had a quick search.
+
+I'm proposing a score card for the security project, to ensure we're doing all that
+we should be doing and identify those areas where we need to improve. I've based this
+on the [Project Team Guide](http://docs.openstack.org/project-team-guide)
+
+Requirement | Status | Notes |
+------------|--------|-------|
+Open Code | Achieved | All code in git and licensed appropriately
+Open Design | Achieved | All design is open to the public, conducted at summits etc
+Open Development | Achieved | We follow standard OpenStack best practice
+Open Community | Needs Improvement | We have a gap around the mailing lists that we need to address
```

Comment:
This is a bit of an understatement. Even mentioned by other members on the mailing list thread above, the community often consists of private groups of people being emailed. That's actually closing off the community. That said, our IRC channel and meetings are open.